src/main/scala/com/databricks/labs/mosaic/functions/MosaicContext.scala

```diff
@@ -654,19 +672,19 @@ object MosaicContext {
 
     def build(indexSystem: IndexSystem, geometryAPI: GeometryAPI): MosaicContext = {
         instance = Some(new MosaicContext(indexSystem, geometryAPI))
-        context
+        context()
     }
 
-    def context: MosaicContext =
+    def context(): MosaicContext =
```

Comment:
@edurdevic these dont really need () no side effects other than Error throw. 
It may simplify jvm calls from spark so works for me.

Reply:
I added those just to get rid of the warnings that IntelliJ was giving me...